### ISSUE TITLE
error: no such module 'AppTrackingTransparency'

### ISSUE THREAD

**Issue by santomegonzalo** [kmcgill88/admob_flutter#219]
Hi, I just updated to the latest beta version `beta-7` and after cleaning everything and running `pod update` I'm constantly getting the following error:

```
↳
    ** BUILD FAILED **
Xcode's output:
↳
    /Users/Documents/personal/flutter/.pub-cache/hosted/pub.dartlang.org/admob_flutter-1.0.0-beta.7/ios/Classes/SwiftAdmobFlutterPlugin.swift:21:8: error: no such module 'AppTrackingTransparency'
    import AppTrackingTransparency
           ^
    /Users/Documents/personal/flutter/.pub-cache/hosted/pub.dartlang.org/admob_flutter-1.0.0-beta.7/ios/Classes/SwiftAdmobFlutterPlugin.swift:21:8: error: no such module 'AppTrackingTransparency'
    import AppTrackingTransparency
           ^
    /Users/Documents/personal/flutter/.pub-cache/hosted/pub.dartlang.org/admob_flutter-1.0.0-beta.7/ios/Classes/SwiftAdmobFlutterPlugin.swift:21:8: error: no such module 'AppTrackingTransparency'
    import AppTrackingTransparency
           ^
    /Users/Documents/personal/flutter/.pub-cache/hosted/pub.dartlang.org/admob_flutter-1.0.0-beta.7/ios/Classes/SwiftAdmobFlutterPlugin.swift:21:8: error: no such module 'AppTrackingTransparency'
    import AppTrackingTransparency
           ^
    /Users/Documents/personal/flutter/.pub-cache/hosted/pub.dartlang.org/admob_flutter-1.0.0-beta.7/ios/Classes/SwiftAdmobFlutterPlugin.swift:21:8: error: no such module 'AppTrackingTransparency'
    import AppTrackingTransparency
           ^
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description

```

Do you have any idea how to solve this error?

**Comment by kmcgill88** [comment]
Beta 7 requires xcode 12 per the change log

**Comment by kmcgill88** [comment]
I've published `1.0.0` and as stated in the changelog required Xcode 12.
